### PR TITLE
BASIC filtering of String fields via query params

### PR DESCRIPTION
**Pull request by conlini**
support to basic string field filtering. Note this does not look up the schema and throw validation if field is not String

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -449,6 +449,30 @@ func (s *Server) tableReadHandler(w http.ResponseWriter, req *http.Request) {
 	log.Tracef("Getting data from table %s", tablename)
 	startTime := time.Now()
 	stmt :="Select * from "+tablename
+// Extract any filters barring sort
```
Comment (otoolep) [inline server/server.go:452]: Indentation looks off. Did you run this through `go fmt`?

**Comment by otoolep** [comment]
I don't really understand what this PR is about. Can you explain?

**Comment by otoolep** [comment]
OK, I see what you are trying to do.

I don't want to do this -- rqlite is about taking what ever SQLite statement is supplied and running it. If you want this type of functionality, then put `WHERE` in the statement and pass that to rqlite. Don't add `WHERE` inside the code. Make sense?

Thanks for the contribution though. I'm closing this, but let me know if I am missing something.

**Comment by conlini** [comment]
@otoolep  So the motivation behind this is to allow restFull interface with queries and agnostic to the underlying SQL. 
Although the original API with q as a parameter is still available, this allows for resource level filtering on the DB

If that is something that would help, I can re-open the PR?

**Comment by otoolep** [comment]
Oh, sorry this is on your fork. My mistake.

Do what you want. :-)

**Comment by otoolep** [comment]
I'm not sure why I've been granted permissions for this repo. Can you remove me? I do not want to have the possibility of damaging other people's forks.

**Comment by pshail** [comment]
@otoolep  Done.

**Pull request merged**